app/models/non_hmis_assessment.rb

```diff
@@ -58,6 +58,20 @@ def self.limited_assessment_types
     ]
   end
 
+  def self.known_assessments_for_matching
+    {}.merge(IdentifiedClientAssessment.new.for_matching).
+      merge(DeidentifiedClientAssessment.new.for_matching).
+      merge(IdentifiedPathwaysAssessment.new.for_matching).
+      merge(DeidentifiedPathwaysAssessment.new.for_matching).
+      merge(IdentifiedCovidPathwaysAssessment.new.for_matching).
+      merge(DeidentifiedCovidPathwaysAssessment.new.for_matching).
+      merge(IdentifiedPathwaysVersionThree.new(assessment_type: :pathways_2021).for_matching).
+      merge(IdentifiedPathwaysVersionThree.new(assessment_type: :transfer_assessment).for_matching).
+      merge(DeidentifiedPathwaysVersionThree.new(assessment_type: :pathways_2021).for_matching).
+      merge(DeidentifiedPathwaysVersionThree.new(assessment_type: :transfer_assessment).for_matching).
+      freeze
+  end
```

Comment:
I'd like to turn this into something like
```
{}.tap do |hash|
  known_assessment_types.each do |clazz|
    hash.merge(clazz.new.for_matching)
  end
end
```
But, I'm not sure what to do with this `assessment_type` parameter on PathwaysV3...

Reply:
Yeah, that's exactly why I ended up where it is.
It is useful for the class to know which slugs/titles to show for the rule, but because the latest iteration requires an "assessment_type" it all broke down.  Happy to entertain other suggestions.

Reply:
Maybe add a `for_matching` param that default to false, and then `clazz.new(for_matching: true).for_matching`?